lib/card_search_query_builder.rb

```diff
@@ -70,72 +82,64 @@ class CardSearchQueryBuilder
-        'f' => 'cards.faction_id',
-        'faction' => 'cards.faction_id',
-        'format' => 'unified_restrictions.format_id',
-        'g' => 'cards.advancement_requirement',
-        'gains_subroutines' => 'cards.gains_subroutines',
-        'h' => 'cards.trash_cost',
-        'has_global_penalty' => 'unified_restrictions.has_global_penalty',
-        'in_restriction' => 'unified_restrictions.in_restriction',
-        'influence_cost' => 'cards.influence_cost',
-        'interrupt' => 'cards.interrupt',
-        'is_banned' => 'unified_restrictions.is_banned',
-        'is_restricted' => 'unified_restrictions.is_restricted',
-        'is_unique' => 'cards.is_unique',
-        'l' => 'cards.base_link',
-        'link_provided' => 'cards.link_provided',
-        'm' => 'cards.memory_cost',
-        'memory_usage' => 'cards.memory_cost',
-        'mu_provided' => 'cards.mu_provided',
-        'n' => 'cards.influence_cost',
-        'num_printed_subroutines' => 'cards.num_printed_subroutines',
-        'o' => 'cards.cost',
-        'on_encounter_effect' => 'cards.on_encounter_effect',
-        'p' => 'cards.strength',
-        'performs_trace' => 'cards.performs_trace',
-        'recurring_credits_provided' => 'cards.recurring_credits_provided',
-        'restriction_id' => 'unified_restrictions.restriction_id',
-        's' => 'card_subtypes.name',
-        'side' => 'cards.card_side_id',
-        'strength' => 'cards.strength',
-        't' => 'cards.card_type_id',
-        'text' => 'cards.stripped_text',
-        'title' => 'cards.stripped_title',
-        'trash_ability' => 'cards.trash_ability',
-        'trash_cost' => 'cards.trash_cost',
-        'u' => 'cards.is_unique',
-        'universal_faction_cost' => 'unified_restrictions.universal_faction_cost',
-        'v' => 'cards.agenda_points',
-        'x' => 'cards.stripped_text',
+        '_' => 'unified_cards.stripped_title',
+        'additional_cost' => 'unified_cards.additional_cost',
+        'advanceable' => 'unified_cards.advanceable',
+        'advancement_cost' => 'unified_cards.advancement_requirement',
+        'agenda_points' => 'unified_cards.agenda_points',
+        'base_link' => 'unified_cards.base_link',
+        'card_pool' => 'unified_cards.card_pool_ids',
+        'card_subtype' => 'unified_cards.card_subtype_names',
+        'card_type' => 'unified_cards.card_type_id',
+        'cost' => 'unified_cards.cost',
+        'd' => 'unified_cards.side_id',
+        'eternal_points' => 'unified_cards.restrictions_points',
+        'f' => 'unified_cards.faction_id',
+        'faction' => 'unified_cards.faction_id',
+        'format' => 'unified_cards.format_ids',
+        'g' => 'unified_cards.advancement_requirement',
+        'gains_subroutines' => 'unified_cards.gains_subroutines',
+        'h' => 'unified_cards.trash_cost',
+        'has_global_penalty' => 'unified_cards.restrictions_global_penalty',
+        'in_restriction' => 'unified_cards.in_restriction',
```

Comment:
`in_restriction` appears to be either `true` or `null`, making it (I think) impossible to search for cards not in restrictions

Reply:
ooh, good catch!  Fixed.